packages/react-instantsearch-hooks-dom/src/widgets/__tests__/all-widgets.test.tsx

```diff
@@ -0,0 +1,101 @@
+/**
```

Comment:
This refactor seems unrelated to this PR.

Reply:
no, it's required, otherwise "all-components.test.ts" can't be written (jsdom is required for testing library)